tutor/src/api.coffee

```diff
@@ -172,6 +173,11 @@ start = (bootstrapData) ->
   apiHelper RosterActions, RosterActions.load, RosterActions.loaded, 'GET', (id) ->
     url: "/api/courses/#{id}/roster"
 
+  apiHelper StudentIdActions, StudentIdActions.load, StudentIdActions.loaded, 'GET', (courseId) ->
+    url: "/auth/status"
+  apiHelper StudentIdActions, StudentIdActions.save, StudentIdActions.saved, 'PATCH', (courseId, params) ->
```

Comment:
Yea, it'd be preferable to have it in the CurrentUserStore.  I was looking there, but couldn't find anything.